test/functional/settlementAidOfferPage.js

```diff
@@ -425,6 +425,17 @@ settlementAidOfferPage.prototype = Object.create({}, {
       get: function() {
         return element( by.css( '.get-options' ) );
       }
+    },
+    //Feedback
+    feedbackLink: {
+      get: function() {
+        return element( by.css( '.feedback .btn' ) );
+      }
+    },
+    followFeedbackLink: {
+      value: function( otherexpensesamount ) {
```

Comment:
Can this function be similar to `confirmVerification` (line 23)? There's a unneeded parameter being passed, and it doesn't need to return anything.